File: client/src/main/java/eu/cqse/teamscale/jacoco/client/agent/Dump.java
/*-------------------------------------------------------------------------+
|                                                                          |
| Copyright (c) 2009-2018 CQSE GmbH                                        |
|                                                                          |
+-------------------------------------------------------------------------*/
package eu.cqse.teamscale.jacoco.client.agent;

import org.jacoco.core.data.ExecutionDataStore;
import org.jacoco.core.data.SessionInfo;

/** All data received in one dump. */
public class Dump {

	/** The session info. */
	public final SessionInfo info;

	/** The execution data store. */
	public final ExecutionDataStore store;

	/** Constructor. */
	public Dump(ExecutionDataStore store, SessionInfo info) {
		this.info = info;
		this.store = store;
	}

}

File: report-generator/src/main/kotlin/com/teamscale/report/jacoco/dump/Dump.kt
/*-------------------------------------------------------------------------+
|                                                                          |
| Copyright (c) 2009-2018 CQSE GmbH                                        |
|                                                                          |
+-------------------------------------------------------------------------*/
package com.teamscale.report.jacoco.dump

import org.jacoco.core.data.ExecutionDataStore
import org.jacoco.core.data.SessionInfo

/** All data received in one dump.  */
class Dump(
	val info: SessionInfo,
	val store: ExecutionDataStore
)
